app/_data/plugins.yml

```diff
@@ -60,6 +60,9 @@
     - name: AWS Lambda
       info: Invoke and manage AWS Lambda functions from Kong
 
+    - name: OpenWhisk
+      info: Invoke and manage OpenWhisk actions from Kong
```

Comment:
Ya know, now that I look at this, the 
> and manage
part isn't really true, is it? Does this plugin (or the AWS Lambda) plugin offer any functionality beyond
> invoke
?

Not asking you to change this because... marketing!